index.js

```diff
@@ -34,7 +34,7 @@ module.exports = function nodeExternals(options) {
             utils.log(mistake.message);
         });
     }
-    var allowlist = [].concat(options.allowlist || []);
+    var allowlist = [].concat(options.allowlist || [/^webpack\/container\/reference\//]);
```

Comment:
Should `/^webpack\/container\/reference\//` always be present? If so - maybe it's better to have 
```
allowlist = [/^webpack\/container\/reference\//].concat(options.allowlist || []);
```